plugin/trino-opensearch/src/main/java/io/trino/plugin/opensearch/OpenSearchMetadata.java

```diff
@@ -674,5 +870,6 @@ public Optional<TableFunctionApplicationResult<ConnectorTableHandle>> applyTable
 
     private record InternalTableMetadata(SchemaTableName tableName, List<ColumnMetadata> columnMetadata, Map<String, ColumnHandle> columnHandles) {}
 
-    private record TypeAndDecoder(Type type, DecoderDescriptor decoderDescriptor) {}
+    @VisibleForTesting
```

Comment:
Is this annotation required? I see `toTrino` returns this data type and is used in tests, but we're not saving it in variables. OTOH, can this be package-private?

Reply:
yes its package private only and hence added the annotated it.